run.sh

```diff
@@ -64,9 +78,20 @@ solana-fullnode "${args[@]}" &
 fullnode=$!
 
 abort() {
+  set +e
   kill "$drone" "$fullnode"
 }
+trap abort INT TERM EXIT
+
+solana-wallet --keypair "$dataDir"/config/leader-keypair.json airdrop 42
+solana-wallet \
+  --keypair "$dataDir"/config/leader-keypair.json  \
+  create-staking-account "$leaderStakingAccountPubkey" 42
```

Comment:
Not annoyed enough yet to add the `configure-staking-account` options to `create-staking-account`?

Reply:
still in "hack it together mode"